utils/bytes_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
+
+import (
+	"math"
+	"math/rand"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIntSize(t *testing.T) {
```

Comment:
not sure I get this test. We are checking some `math` package quantities, not anything related to our code.
Is it to make sure we have something breaking on platform we do not currently support? Can you clarify?

Reply:
I'll add a comment